packages/react/index.classic.fb.js

```diff
@@ -26,6 +26,8 @@ export {
   useReducer,
   useRef,
   useState,
+  useMutableSource,
```

Comment:
When we land this in www, I suggest prefixing these APIs in the entry point file until we're ready for everyone to  use them. Then tell people to only use it via a module. That way we can change the API without having to run a codemod.